example/windows/win32_window.cc

```diff
@@ -17,12 +16,26 @@ constexpr int kBaseDpi = 96;
 
 constexpr LPCWSTR kClassName = L"CLASSNAME";
 
+using EnableNonClientDpiScaling_ = BOOL __stdcall(HWND hwnd);
+
 // Scale helper to convert logical scaler values to physical using passed in
 // scale factor
 int Scale(int source, double scale_factor) {
   return static_cast<int>(source * scale_factor);
 }
 
+// Dynamically loads the |EnableNonClientDpiScaling| from the User32 module.
+// Appended with FDE to differentiate from the win32 API.
```

Comment:
Let's just give it a different name instead. E.g., `EnableFullDpiSupportIfAvailable` or something along those lines.